bitcoin/Cargo.toml

```diff
@@ -79,3 +79,6 @@ required-features = ["std", "rand-std", "bitcoinconsensus"]
 
 [[example]]
 name = "sighash"
+
+[lints.rust]
+unexpected_cfgs = { level = "deny", check-cfg = ['cfg(bench)', 'cfg(fuzzing)', 'cfg(kani)', 'cfg(mutate)', 'cfg(rust_v_1_60)'] }
```

Comment:
> `rust_v_1_60`

Shouldn't this be set in your [`build.rs`](https://github.com/rust-bitcoin/rust-bitcoin/blob/master/bitcoin/build.rs#L31) where the cfg is defined?